app/tasks.py

```diff
@@ -68,17 +74,42 @@ def execute(impl_url, impl_data, impl_language, transpiled_qasm, input_params, t
             result.complete = True
             db.session.commit()
         app.logger.info('Start transpiling...')
-        try:
-            transpiled_circuits = transpile(circuits, backend=backend, optimization_level=optimization_level)
-        except TranspilerError:
-            result = Result.query.get(job.get_id())
-            result.result = json.dumps({'error': 'too many qubits required'})
-            result.complete = True
-            db.session.commit()
 
-    app.logger.info('Start executing...')
-    job_manager = IBMQJobManager()
-    job_result = job_manager.run(transpiled_circuits, backend=backend)
+        if noise_model:
+            noisy_qpu = get_qpu(kwargs, noise_model)
+            noise_model = NoiseModel.from_backend(noisy_qpu)
+            properties = noisy_qpu.properties()
+            configuration = noisy_qpu.configuration()
+            coupling_map = configuration.coupling_map
+            basis_gates = noise_model.basis_gates
+            transpiled_circuits = transpile(circuits, noisy_qpu)
```

Comment:
there should be also a try catch block around the transpile statement as in line 100

Reply:
in case too many qubits are needed